File: frontend/types/bitacora.ts
export interface BitacoraEntry {
  uuid: string;
  id_incidente: string;
  id_usuario: string;
  comentario: string;
  eliminado: boolean;
  created_at: string | null;
  usuario_nombre: string | null;
}

export interface BitacoraCreate {
  comentario: string;
}

File: backend/app/schemas/bitacora.py
from datetime import datetime
from uuid import UUID

from pydantic import BaseModel, Field


class BitacoraCreate(BaseModel):
    comentario: str = Field(min_length=1)


class BitacoraOut(BaseModel):
    uuid: UUID
    id_incidente: UUID
    id_usuario: UUID
    comentario: str
    eliminado: bool = False
    created_at: datetime | None = None
    usuario_nombre: str | None = None
